### PR TITLE
Update plotly

### PR DESCRIPTION
**Pull request by esloch**
- This PR is open as a work in progress for discussions and suggestions of changing the styles of graphics and for validating the code.
- Updated .plotly to version 4.0.0, few background, grid_colors, font_colors, lines, dashes and borders color adjustments.

### CODE REVIEW

**Comment by esloch** [comment]
Screenshot_20190803_054500
Screenshot_20190803_054515
Screenshot_20190803_054538

**Comment by fccoelho** [comment]
Good job @esloch ! I think that we should merge it and update infostaging. Then, from the live site it will be easier to check for issues regarding data correcteness and presentation.

File: AlertaDengue/requirements.txt [hunk 1]
```diff
@@ -23,7 +23,6 @@ rasterio
 geopy
 geopandas
 sh
-plotly==2.5.1
-#cufflinks
+plotly==4.0.0rc1
```
[the remark this answers is not in the document]
Comment (esloch) [reply AlertaDengue/requirements.txt]: Is ready!

File: AlertaDengue/dados/charts.py [hunk 2]
```diff
@@ -200,14 +271,15 @@ def create_climate_chart(
...
 
-        return _plot_html(
-            figure_or_data=figure,
-            config={},
-            validate=True,
-            default_width='100%',
-            default_height=500,
-            global_requirejs='',
-        )[0]
+        # return _plot_html(
```
[the remark this answers is not in the document]
Comment (esloch) [reply AlertaDengue/dados/charts.py]: Is ready!

File: AlertaDengue/environment.yml [hunk 3]
```diff
@@ -1,6 +1,7 @@
 name: alertadengue
 channels:
   - conda-forge
+  - plotly/label/test
```
[the remark this answers is not in the document]
Comment (esloch) [reply AlertaDengue/environment.yml]: Is ready!

**Comment by esloch** [comment]
@xmnlab, lightgrey result very dark however, then change to a color in #rgb.
Thanks!

**Pull request merged**